packages/client-proxy/internal/edge/pool/edge-pool.go

```diff
@@ -192,19 +192,19 @@ func (p *EdgePool) removeNode(ctx context.Context, node *EdgeNode) error {
 	_, childSpan := p.tracer.Start(ctx, "remove-edge-node")
 	defer childSpan.End()
 
-	p.logger.Info("Edge node connection is not active anymore, closing.", zap.String("node_id", node.ServiceId))
+	p.logger.Info("Edge node Connection is not active anymore, closing.", zap.String("node_id", node.ServiceId))
```

Comment:
uppercased this by mistake during some refactor?

Reply:
yes, fixed thx